pyproject.toml

```
@@ -1,10 +1,24 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "RUFAS"
+version = "0.9.2"
+description = "Ruminant Farm Systems Model"
+readme = "README.md"
+authors = [
+{"name" = "Kristan Foster Reed", "email" = "Kristan.Reed@usda.gov"},
+]
+requires-python = ">=3.12"
+
+
```

```suggestion

[project.urls]
"Homepage = "https://rufas.org"
"Source" = "https://github.com/RuminantFarmSystems/MASM"
```
I'm not 100% sure this is the right way to add urls, but I think it would be good to include them

---

https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#urls